tools/abi-check.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh -e
 status=0
 # Dump the current ABI
-abi-dumper ./build/libyang.so -o ./build/libyang.dump -lver "$(PKG_CONFIG_PATH=./build pkg-config --modversion libyang)"
+LC_ALL=C.UTF-8 PATH=/snap/bin:$PATH abi-dumper ./build/libyang.so -o ./build/libyang.dump -lver "$(PKG_CONFIG_PATH=./build pkg-config --modversion libyang)" -public-headers ./src -public-headers ./build/src
```

Comment:
Why the snap is needed here? The ABI tools are installed via apt in Travis, so I guess they will be in the standard location.

Reply:
ahh, the universal-ctags are installed via snap

Reply:
Unfortunately, the `universal-ctags` package is only available from Ubuntu eoan and up. The `abi-dumper` produces slightly better output when it can parse the public headers, so I wanted to have the `ctags` available for it.  When Travis CI starts to provide Ubuntu focal, this can go away and the packaged version can be used.